sn_transfers/src/cashnotes/spend_reason.rs

```diff
@@ -40,7 +40,7 @@ impl SpendReason {
     }
 
     pub fn create_reward_tracking_reason(input_str: &str) -> Result<Self> {
-        let input_pk = crate::NETWORK_ROYALTIES_PK.public_key();
+        let input_pk = crate::FOUNDATION_PK.public_key();
```

Comment:
the foundation_key has already having large amount of fund transfered in, from the genesis,
hence chose `royalty_key` to forward payment to
actually, maybe we'd better to have `another` key-pair to forward to ?